<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>设置对象</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rbtnListSrc">
           <property name="text">
            <string>listSource</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnListWidget">
           <property name="text">
            <string>listWidget</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtntreeWidget">
           <property name="text">
            <string>treeWidget</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnTableWidget">
           <property name="text">
            <string>tableWidget</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>拖放参数设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkAcceptDrops">
           <property name="text">
            <string>acceptDrops</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="chkDragEnable">
           <property name="text">
            <string>dragEnable</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>dragDropMode</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboMode">
           <item>
            <property name="text">
             <string>NoDragDrop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DragOnly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DropOnly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DragDrop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InternalMove</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>defaultDropAction</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboAction">
           <item>
            <property name="text">
             <string>CopyAction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MoveAction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LinkAction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IgnoreAction</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>44</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>listSource</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="listSrc">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>剪切</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/200.bmp</normaloff>:/icons/images/200.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>复制</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/202.bmp</normaloff>:/icons/images/202.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>粘贴</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/204.bmp</normaloff>:/icons/images/204.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>粗体</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/500.bmp</normaloff>:/icons/images/500.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>斜体</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/502.bmp</normaloff>:/icons/images/502.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>下划线</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/504.bmp</normaloff>:/icons/images/504.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左对齐</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/508.bmp</normaloff>:/icons/images/508.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中间对齐</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/510.bmp</normaloff>:/icons/images/510.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右对齐</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/512.bmp</normaloff>:/icons/images/512.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>红色</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/718.bmp</normaloff>:/icons/images/718.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>绿色</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/724.bmp</normaloff>:/icons/images/724.bmp</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蓝色</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/images/728.bmp</normaloff>:/icons/images/728.bmp</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>listWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>treeWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>编辑</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>编辑</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>格式</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>tableWidget</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="rowCount">
           <number>6</number>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>19</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>25</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
